packages/swing-store/src/snapStore.js

```diff
@@ -75,27 +83,35 @@ const sink = () => {};
  * @returns {SnapStore}
  */
 export function makeSnapStore(
+  db,
   root,
   {
     createReadStream,
     createWriteStream,
     measureSeconds,
-    fsync,
-    rename,
-    resolve: pathResolve,
     stat,
     tmpFile,
     tmpName,
     unlink,
   },
   { keepSnapshots = false } = {},
 ) {
+  db.exec(`
+    CREATE TABLE IF NOT EXISTS snapshots (
+      hash TEXT,
+      snapshot BLOB,
```

Comment:
Should we name this column `compressedSnapshot`? I'd like to make it clear what the contents are.